api/pkg/db/initializer/initiailizer.go

```diff
@@ -94,3 +98,49 @@ func (i *Initializer) addCatalogs() error {
 	}
 	return nil
 }
+
+func (i *Initializer) addUsers() error {
+
+	db := i.db
+	// Checks if tables exists
+	if !db.HasTable(&model.User{}) || !db.HasTable(model.Scope{}) {
```

Comment:
Can't we directly use `i.db` instead of having a new variable

Reply:
`db` would be fine I guess, make the code clean :D